src/distribution/triangular.rs

```diff
@@ -0,0 +1,241 @@
+use distribution;
+use source::Source;
+
+/// An triangualr distribution.
+#[derive(Clone, Copy)]
+pub struct Triangular {
+    a: f64,
+    b: f64,
+    c: f64,
+}
+
+impl Triangular {
+    pub fn new(a: f64, b: f64, c: f64) -> Self {
```

Comment:
Let’s add a description here, mentioning also that it panics if the arguments are invalid. Have a look at `new` of any other distribution.